src/thor/trace_route_action.cc

```diff
@@ -40,6 +40,7 @@ constexpr size_t kMatchResultsIndex = 2;
 constexpr size_t kTripLegIndex = 3;
 
 void add_path_edge(valhalla::Location* l, const meili::MatchResult& m) {
+  l->mutable_path_edges()->Clear();
```

Comment:
for the route matcher code path, we use loki to find edge candidates. i need to verify but i think that is where these are coming from. at any rate we do not want them here. we only want the edge candidate that the map matched path is actually using. in other words we cant just append.